pontoon/settings/base.py

```diff
@@ -789,10 +790,12 @@ def _default_from_email():
     "django.contrib.staticfiles.finders.FileSystemFinder",
     "django.contrib.staticfiles.finders.AppDirectoriesFinder",
 )
-STATICFILES_DIRS = [
+
+STATICFILES_DIRS = (
+    ("badges", "pontoon/contributors/static/img"),
```

Comment:
Why do we need this change?

Reply:
As I understand it, Django currently only serves static files from `base/static`. I believe adding this change allows the 3 badge svg files to also be served in our app.

The tuple is just to for [better readability](https://docs.djangoproject.com/en/5.1/ref/settings/#prefixes-optional).

Is there a better way of doing this?

Reply:
We use https://docs.djangoproject.com/en/5.1/ref/contrib/staticfiles/#collectstatic to collect the static files.

You can check https://docs.djangoproject.com/en/5.1/ref/contrib/staticfiles/#findstatic to check the paths:

```bash
make shell
./manage.py findstatic img
```